recipes/re2/all/CMakeLists.txt

```diff
@@ -1,11 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(cmake_wrapper)
 
-if(EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-  include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-else()
-  include(conanbuildinfo.cmake)
-endif()
+include(conanbuildinfo.cmake)
```

Comment:
I feel like we should open an issue discussing this...

Please don't just remove this as it breaks workflows

Reply:
I can restore this, but a lot of recipes in CCI don't try to include `"${CMAKE_BINARY_DIR}/conanbuildinfo.cmake"`, is it a bad thing? What kinds of workflow need this when using CCI recipes?

Reply:
Development flow

Many recipes probably don't have it because Bincrafters recipes templates didn't had it always or because people don't know about this workflow or don't care

https://github.com/bincrafters/templates/commit/0126b3e9b6856157c692e62a6f6ab130a8c62440

Reply:
So what you say implies that we should add this in all CCI recipes (also in test_package?)?

Reply:
I think just leave it as it was before. it's not right time to touch this code, as soon we will need to touch it again - we're trying to move away from too intrusive approaches, requiring injections of conan commands into CMakeLists.txt.

Reply:
ok, fixed